### ISSUE TITLE
Another way to get the Polymer documention ?

### ISSUE THREAD

**Issue by Krazitchek** [Polymer/docs#1274]
Hello, isn't it possible to get the documention for an offline usage in a simple way, like downloading an archive file, or a PDF, or something else easily exploitable without installing a lot of tools, cloning repository, fighting with command line, etc, etc...

Personaly, the way indicated (clone repo, install ruby, jekyll, other stuffs, modifying path...) does not work form me, i have multiple errors about ruby/gem and other stuff i don't understand. Howerver, i respect the procedure described step by step !

An easily downolable documentation would be great.

**Comment by ebidel** [comment]
Hi @Krazitchek. Those instructions are mainly for contributors to the docs. Offline/downloadable docs are not high priority for us since we iterated fairly quickly. You could always download this repo as a .zip from Github and read over the markdown files. I realize that's not formatted or ideal, but it would get you something offline.

**Comment by CemraJC** [comment]
@ebidel, I have downloaded the Polymer repo for offline use (Internet sucks where I am) and I am attempting to build the polymer docs to be formatted (cause the raw markdown is simply unreadable)

**However!** Once I get all the dependencies and gems and whatnot, the grunt task to run the appserver fails (This is a Google App Engine issue). I have followed copious amounts of tutorials, I have only Python 2.7.3 installed, but *still* I cannot build the Docs and view them. I have been trying for weeks...

So - **my question is this**: Is there any alternative to using the Google App Engine SDK for Python? Maybe something I could do to move away from GAE - because it has some issues I cannot resolve.

Sorry for the length :smiley: 

P.S. I am developing (groan) from Windows 7 *I have no other option*
